src/sentry/static/sentry/app/utils/withProjects.tsx

```diff
@@ -9,10 +9,12 @@ import {Project} from 'app/types';
 
 type InjectedProjectsProps = {
   projects: Project[];
+  loadingProjects?: boolean;
 };
 
 type State = {
   projects: Project[];
+  loadingProjects: boolean;
```

Comment:
Is this being used?

Reply:
woops I misnamed it :/ but this loading field should be populated from ProjectsStore.getState()